proposals/pr002-hunkspace.md

```diff
@@ -0,0 +1,544 @@
+# CLDR "Hunkspace"
+
+2020-07-09
+
+Author: [filmil (Filip Filmar) · GitHub][ff]
+
+*tl;dr:* Generalize the [ICU4X Data Provider][dp] key space to include
+localized data from sources other than [CLDR][cldr] More detail is in the
+[Detailed Proposal](#h.us1rqruwdp1h) section, after the proposal is motivated
+by a few use cases.
+
+This document proposes a global, shared *hunkspace* (better names always
+welcome) of localized "[hunk][hunk]" data for the ICU4X Data Provider.  It
+doesn't necessarily affect the ICU4X data provider implementation, but does
+affect the data model.   Specifically the proposal is to make the data model
+more general and open to independent third party extensions.
+
+> The terminology proposed in [ICU4X Locale Data Pipeline][icudp] is used.  Any
+> inconsistent word use is a bug, and should be fixed.
+
+[cldr]: https://cldr.unicode.org
```

Comment:
nit: broken link [https://cldr.unicode.org](https://cldr.unicode.org) fails but [http://cldr.unicode.org](http://cldr.unicode.org) does not. Might just be on my end though (tested on ChromeOS, MacOS, WiFi, and LTE)

Reply:
Thanks for catching this! Fixed.